src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/RazorCompletionEndpoint.cs

```diff
@@ -267,6 +268,7 @@ internal static bool TryConvert(RazorCompletionItem razorCompletionItem, out Com
                         {
                             Label = razorCompletionItem.DisplayText,
                             InsertText = razorCompletionItem.InsertText,
+                            InsertTextFormat = InsertTextFormat.PlainText,
```

Comment:
Hmm why add this?

Reply:
If this was done just to be explicit then should add this in other place where we new up a `CompletionItem`.

Reply:
Yeah, I was just trying to be explicit, fair duce though, I'll remove this one.